### PR TITLE
fix: gc checking if pantahub.config exists

### PR DESCRIPTION
**Pull request by anibalportero**
This is subsituted by creating a file /storage/config/mounted after mounting /storage. That file is then checked before getting storage info for GC and logs.

### CODE REVIEW

File: paths.h [hunk 1]
```diff
@@ -87,6 +87,7 @@ void pv_paths_storage_trail_pvr_file(char *buf, size_t size, const char *rev,
 
 #define UNCLAIMED_FNAME "unclaimed.config"
 #define PANTAHUB_FNAME "pantahub.config"
+#define MOUNTED_FNAME "mounted"
```
Comment (asac) [inline paths.h:90]: lets name it .pv_mounted .... same we probably would want to add to all disks we mount (e.g. crypt disks)... and later then also gather space of those disks of course...

**Review by asac — CHANGES_REQUESTED** [review]

File: paths.h (hunk 1 above)
Comment (anibalportero) [reply paths.h]: done

File: storage.c [hunk 2]
```diff
@@ -220,7 +220,7 @@ static struct pv_storage *pv_storage_new()
 	struct statfs buf;
 	struct pv_storage *this;
 
-	pv_paths_storage_config_file(path, PATH_MAX, PANTAHUB_FNAME);
+	pv_paths_storage_file(path, PATH_MAX, PVMOUNTED_FNAME);
```
Comment (asac) [inline storage.c:223]: why this change?

Comment (anibalportero) [reply storage.c:223]: well, I imagined if we are going to use .pvmounted for disks too why not storing that in /storage instead of /storage/config

**Review by asac — APPROVED** [review]
